tasks/eth/027-holocene-system-config-upgrade-and-init-multi-chain/VALIDATION.md

```diff
@@ -0,0 +1,140 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## Nested Safe State Overrides and Changes
+
+This task is executed by the nested 2/2 `ProxyAdminOwner` Safe. Refer to the
+[generic nested Safe execution validation document](../../../NESTED-VALIDATION.md)
+for the expected state overrides and changes.
+
+The `approvedHashes` mapping **key** of the `ProxyAdminOwner` that should change during the simulation is
+- Council simulation: `0xfae39fd96196a971566654dbfb0f11d7b98980213f6410051ccbf81f7481039`
+- Foundation simulation: `0x0595de264ad16565584215810251811f9e56d429c8fd67def448041126dbe920`
+
+calculated as explained in the nested validation doc:
+
+```sh
+cast index address 0xc2819DC788505Aac350142A7A707BF9D03E3Bd03 8 # council
+# 0xaaf2b641eaf0bae063c4f2e5670f905e1fb7334436b902d1d880b05bd6228fbd
+cast index bytes32 0xc94a72f6ae0da3b87a2ecb6e6ce52608c8a9d722a1b5f5f46ed9cc70a54f8a03 0xaaf2b641eaf0bae063c4f2e5670f905e1fb7334436b902d1d880b05bd6228fbd
+# 0xfae39fd96196a971566654dbfb0f11d7b98980213f6410051ccbf81f74810394
+```
+
+```sh
+cast index address 0x847B5c174615B1B7fDF770882256e2D3E95b9D92 8 # foundation
+# 0x13908ba1c0e379ab58c6445554ab471f3d4efb06e3c4cf966c4f5e918eca67bd
+cast index bytes32 0xc94a72f6ae0da3b87a2ecb6e6ce52608c8a9d722a1b5f5f46ed9cc70a54f8a03 0x13908ba1c0e379ab58c6445554ab471f3d4efb06e3c4cf966c4f5e918eca67bd
+# 0x0595de264ad16565584215810251811f9e56d429c8fd67def448041126dbe920
+```
+
+## State Changes
+
+### `0x229047fed2591dbec1eF1118d64F7aF3dB9EB290` (`SystemConfigProxy` for op-mainnet)
```

Comment:
Not related to the validation but just to make sure `op-mainnet` and `mode-mainnet` and `zora-mainnet` are not updating the scalarVersion to `1` and this is intended right and will keep the `0` value.

Reply:
Thanks correct the slot 0x66 is not changed!
I get quickly confuse about it but that completely make sense sorry for the naive question haha!